src/transaction/transactionApi.ts

```diff
@@ -45,6 +45,10 @@ export class TransactionApi extends Base {
     });
   }
 
+  async listTransactions() {
+    return await this.get<Transaction>('transactions');
```

Comment:
note: may need a different type here, since sync and async transactions return slightly different types

Reply:
The backend work has now been merged, but still need a new type here.